src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -230,16 +230,23 @@ default String getEffectiveAvatarUrl()
      * Represents the bit offsets used by Discord for public flags
      */
     enum Flags{
-        NONE(            0, "None"),
-        EMPLOYEE(        1, "Discord Employee"),
-        PARTNER(         2, "Discord Partner"),
-        HYPESQUAD(       4, "HypeSquad Events"),
-        BUG_HUNTER_1(    8, "Bug Hunter Level 1"),
-        BRAVERY(        64, "HypeSquad House Bravery"),
-        BRILLIANCE(    128, "HypeSquad House Brilliance"),
-        BALANCE(       256, "HypeSquad House Balance"),
-        SYSTEM(       4096, "System"),
-        BUG_HUNTER_2(16384, "Bug Hunter Level 2");
+        NONE(                      0, "None"),
+        STAFF(                     1, "Discord Employee"),
+        PARTNER(                   2, "Discord Partner"),
+        HYPESQUAD(                 4, "HypeSquad Events"),
+        BUG_HUNTER_LEVEL_1(        8, "Bug Hunter Level 1"),
+        // Bravery
+        HYPESQUAD_ONLINE_HOUSE_1( 64, "HypeSquad Online House Bravery"),
+        // Brilliance
+        HYPESQUAD_ONLINE_HOUSE_2(128, "HypeSquad Online House Brilliance"),
+        // Balance
+        HYPESQUAD_ONLINE_HOUSE_3(256, "HypeSquad Online House Balance"),
+        PREMIUM_EARLY_SUPPORTER( 512, "Early Supporter"),
+        TEAM_USER(              1024, "Team User"),
+        SYSTEM(                 4096, "System User"),
+        BUG_HUNTER_LEVEL_2(    16384, "Bug Hunter Level 2"),
+        VERIFIED_BOT(          65536, "Verified Bot"),
+        VERIFIED_DEVELOPER(   131072, "Verified Developer");
```

Comment:
Instead of using raw ints, please use the offsets and calculate the raw value in the constructor.

Reply:
Was changed int https://github.com/DV8FromTheWorld/JDA/pull/3